source/slang/slang-ir-layout.cpp

```diff
@@ -443,6 +443,38 @@ struct NaturalLayoutRules : IRTypeLayoutRules
     }
 };
 
+struct ConstantBufferLayoutRules : IRTypeLayoutRules
+{
+    ConstantBufferLayoutRules()
+    {
+        ruleName = IRTypeLayoutRuleName::D3DConstantBuffer;
+    }
+
+    virtual IRSizeAndAlignment alignCompositeElement(IRSizeAndAlignment elementSize)
```

Comment:
Is `alignCompositeElement` ever used? If not we should just change its behavior in-place wihtout adding another variant.

Reply:
The variant can be removed 👍